### DISCUSSION TITLE
Capitole du libre

### DISCUSSION THREAD

**Discussion by issantos** [Stellarium/stellarium#2626] [General]
Every year (except during covid) in Toulouse, France, there is a convention on libre and open source called the Capitole du Libre (more info here: https://capitoledulibre.org/). This year it will be November 19th-20th. 

I would like to hold a stand to present Stellarium during this event, so that more people learn about Stellarium, and possibly gain new contributors. This leads to my two questions:

- Is there anyone in the French Stellarium community that would like to hold the stand with me?
- Are there any posters/visuals/etc that have been done for similar events that could be re-used?

**Comment by alex-w** [comment]
@xalioth @gzotti?

**Reply by gzotti** [reply]
Nice idea! I will not be around, though. 
I have 2 presentation videos on YouTube, linked from the stellarium.org website ("News" panel), and an open-access paper from last year which you could print.
You could probably also run some existing scripts to show off some features, Messier marathon or other things.

**Comment by issantos** [comment]
The Stellarium stand at the Capitole du Libre in Toulouse last weekend was a huge success. We were three people holding the stand and there constantly were people coming to ask us questions about Stellarium all weekend long. Many visitors didn't know the software at all and most of those who did learned that it is open source. Several long-time Stellarium users discovered functions they didn't know about. A few people asked how to make a donation to the project.
